resources/TrackingTabComponents/TrackingSkyPlotTab.qml

```diff
@@ -302,7 +302,7 @@ Item {
                         Label {
                             color: 'black'
                             text: '` + labels[idx][jdx] + `'
-                            visible: (!polarChartWidthChanging && labelsVisible && seriesEntry.visible)
+                            visible: !polarChartWidthChanging
```

Comment:
since label is destroyed on timer, we dont need these visibility fields

Reply:
i agree this can be removed but I think your reasoning might be wrong (the logic earlier in the control flow ensures that at this point in the code `labelsVisible`  & `seriesEntry.visible` are always `true`.

Reply:
yea the main issue was inserting object calling seriesEntry field which isn't declared globally causing the ReferenceError. im assuming it wanted to only use one label each (but coords not updated) but theres timer to clean it up whenever spawned so it didn't really make sense. 

i.e. if we dont have a timer to clean up labels then we would need visbility to refer to the series